frontend/public/actions/k8s.ts

```diff
@@ -265,8 +265,7 @@ export const watchK8sList = (
         // eslint-disable-next-line no-console
         console.log('WS closed abnormally - starting polling loop over!');
         const ws = WS[id];
-        const timedOut = true;
-        ws && ws.destroy(timedOut);
+        ws && ws.destroy();
```

Comment:
Oh damn, there was an instance... not sure how I missed that. But yup, that is not doing anything, good to remove.